src/Tribe/Main.php

```diff
@@ -3054,6 +3057,10 @@ public function publishAssociatedTypes( $post_id, $post ) {
 			remove_action( 'save_post_' . self::VENUE_POST_TYPE, array( $this, 'save_venue_data' ), 16, 2 );
 			remove_action( 'save_post_' . self::ORGANIZER_POST_TYPE, array( $this, 'save_organizer_data' ), 16, 2 );
 
+			// Remove any "preview" venues and organizers (duplicates) attached to this event.
+			$this->remove_preview_venues( $post_id, true );
```

Comment:
can you clarify what you mean @gustavo? the hook is "hooked" to the publish action in a method that is itself running `plugins_loaded`, but this `remove_preview_venues` function itself doesn't run on `plugins_loaded` -- right? apologies if i'm missing something here -- this function runs on the `publish_{post_type}` action *only*